### PR TITLE
KidSmiley

### PR DESCRIPTION
**Pull request by refun07**

### CODE REVIEW

File: home/templates/home/about.html [hunk 1]
```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
```
Comment (tareqobaida) [inline home/templates/home/about.html]: Follow the format from other html file. You dont have to define header and footer. Just look at other files as examples.

File: home/templates/home/footer.html [hunk 2]
```diff
@@ -2,8 +2,9 @@
 <footer>
     <div class="content container-fluid">
```
Comment (tareqobaida) [inline home/templates/home/footer.html:3]: Use social icons instead of images in footer.

**Review by tareqobaida — CHANGES_REQUESTED** [review]

File: home/templates/home/about.html (hunk 1 above)
Comment (refun07) [reply home/templates/home/about.html]: I will looking forward into the issues.

File: home/templates/home/footer.html (hunk 2 above)
Comment (refun07) [reply home/templates/home/footer.html:3]: Yes.please check the latest version.It's up and running with social icon!

**Review by tareqobaida — APPROVED** [review]

**Pull request merged**